src/cmd/cli/command/mcpClient.go

```diff
@@ -0,0 +1,128 @@
+package command
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/DefangLabs/defang/src/pkg/term"
+)
+
+// ClientProcessMap maps client names to their process names
+var ClientProcessMap = map[string]string{
+	"claude":   "Claude",
+	"windsurf": "Codeium",
+	"cursor":   "Cursor",
+	"vscode":   "Code",
+}
+
+// IsClientRunning checks if a client process is currently running
+func IsClientRunning(client string) bool {
```

Comment:
This feels very brittle. We might match unexpected processes with matching substrings and then attempt to kill them. I would suggest we avoid this feature for now and find a simpler compromise until we can think through the risks here more carefully.